ladybug/windrose.py

```diff
@@ -0,0 +1,156 @@
+"""Ladybug windrose"""
+from ladybug.epw import EPW
+
+
+class WindRose(object):
+    """
+    Draws the Ladybug windrose
+    Usage:
+    """
+    __slots__ = ("epw", "HOYs", "annualHourlyData",
+                 "windCondition", "dataCondition",
+                 "numOfDirections", "centerPoint",
+                 "scale", "maxFrequency")
+
+    def __init__(self,
+                 epw_filePath,
+                 HOYs=range(0, 8760),
+                 annualHourlyData=None,
+                 windCondition=None,
+                 dataCondition=None,
+                 ):
+        """
+        Instantiates the Ladybug WindRose Object
+        """
+        self.epw = EPW(epw_filePath)
+        self.HOYs = HOYs
+        self.annualHourlyData = annualHourlyData
+        self.windCondition = self.check_condition(windCondition)
+        self.dataCondition = self.check_condition(dataCondition)
+
+    @staticmethod
+    def check_condition(condition):
+        """
+        Checks whether the conditional statement has
+        the variable 'x'
+        Args:
+            condition: A list of conditional statements
+        Returns:
+            None: if condition is None or the variable 'x' is not \
+            in the conditional statement
+            condition: The same list of conditional statements provided
+            as arguments to the function
+
+        """
+        if condition is None:
+            return None
+        if isinstance(condition, str):
+            if "x" in condition.lower():
+                return condition
+            else:
+                print ("The conditon shall be a valid string with" +
+                       " a variable 'x' in it.")
+        else:
+            xcount = 0
+            for statement in condition:
+                if "x" in statement.lower():
+                    xcount += 1
+            if len(condition) == xcount:
+                return condition
+            else:
+                print ("All conditions shall be a valid string with" +
+                       "a variable 'x' in it.")
+                return None
+
+    def data_to_HOY(self):
```

Comment:
The same here. The point of using DataCollection class is not to rewrite methods like this.